.github/workflows/release.yml

```diff
@@ -0,0 +1,246 @@
+          echo "pkg version:  ${{ needs.tag.outputs.PKG_VER }}"
+
+  build-linux-all:
+    runs-on: parity-large
+    needs: [create-release, tag]
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: install linux deps
+        run: |
+          sudo apt-get update && sudo apt-get install -y cmake ninja-build \
+          curl git libssl-dev pkg-config clang lld nodejs npm musl
+
+      - name: Install Rust stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          components: rust-src
+          target: wasm32-unknown-emscripten
+
+      - name: versions
+        run: |
+          rustup show
+          cargo --version
+          cmake --version
+          echo "bash:" && bash --version
+          echo "ninja:" && ninja --version
+          echo "clang:" && clang --version
+
+      - name: build revive-llvm
+        run: make install-llvm-builder
+
+      # Restore cache
+
+
+      - name: llvm-musl-cache restore
+        id: llvm-musl-cache
+        uses: actions/cache/restore@v4
+        with:
+          path: target-llvm/musl/target-final
+          key: llvm-linux-musl-${{ hashFiles('crates/solidity/**') }}
+
+      - name: llvm-emscripten-cache restore
+        id: llvm-emscripten-cache
+        uses: actions/cache/restore@v4
+        with:
+          path: |
+            target-llvm/emscripten/target-final
+            emsdk
+          key: llvm-linux-emscripten-${{ hashFiles('crates/solidity/**') }}
+
+      # Build LLVM
+
+
+      - name: Build musl LLVM
+        if: steps.llvm-musl-cache.outputs.cache-hit != 'true'
+        run: |
+          revive-llvm --target-env musl clone
+          revive-llvm --target-env musl build --llvm-projects lld --llvm-projects clang
+
+      - name: Build emscripten LLVM
+        if: steps.llvm-emscripten-cache.outputs.cache-hit != 'true'
+        run: |
+          revive-llvm --target-env emscripten clone
+          source emsdk/emsdk_env.sh
+          revive-llvm --target-env emscripten build --llvm-projects lld
+
+      # Save cache
+
+
+      - name: llvm-musl-cache save
+        if: steps.llvm-musl-cache.outputs.cache-hit != 'true'
+        uses: actions/cache/save@v4
+        with:
+          path: target-llvm/musl/target-final
+          key: llvm-linux-musl-${{ hashFiles('crates/solidity/**') }}
+
+      - name: llvm-emscripten-cache save
+        if: steps.llvm-emscripten-cache.outputs.cache-hit != 'true'
+        uses: actions/cache/save@v4
+        with:
+          path: |
+            target-llvm/emscripten/target-final
+            emsdk
+          key: llvm-linux-emscripten-${{ hashFiles('crates/solidity/**') }}
+
+      # Build revive
+
+      - name: download solc
+        run: |
+          curl -o solc https://github.com/ethereum/solidity/releases/download/v0.8.28/solc-static-linux
+          chmod +x solc
+          echo "$PWD" >> $GITHUB_PATH

```

Comment:
This is technically not required for builds.

However, executing a basic tests using the release build, like compiling a flipper contract, would be good. Can you please add such a test? It can be really simple, i.e. running `resolc --bin crates/integration/contracts/flipper.sol`, expecting the PVM magic bytes `50564d` in the output and expecting a return value of 0. Same for the Wasm build. It's unlikely but should the release workflow for some reason produce defective artifacts, without the test, we will only find it out the annoying way.